centos-ci/libtask.sh

```diff
@@ -18,9 +18,6 @@ prepare_job() {
 
     . ~/rsync-password.sh
 
-    # Ensure we're operating on a clean base
-    (cd ${buildscriptsdir} && git clean -dfx && git reset --hard HEAD)
```

Comment:
The `sig-atomic-buildscript` on which this runs comes from the rsync of the job workspace to the node. But since all the jobs have their workspaces cleaned on startup, their clone of `sig-atomic-buildscripts` are clean.

I just realized the commit message is wrong. It should say `run-treecompose`, not `run-tree-smoketest`. I.e. the treecompose jobs will rsync a `sig-atomic-buildscripts` repo in which the `releases.yml` file is modified, but then that gets wiped out by the `git reset` here.

Alternatively, we can commit the modification before rsync'ing if you'd like?